README.md

```diff
@@ -357,6 +358,20 @@ TwilioVoice.audioDevice = audioDevice;
 TVOCall *call = [TwilioVoice connectWithOptions:connectOptions delegate:self];
 ```
 
+### <a name="preferred-audio-codec"></a>Preferred Audio Codec
+In Voice iOS 3.X, you can provide your preferred audio codec in the `TVOConnectOptions` and the `TVOAcceptOptions`.
```

Comment:
Please add the following:

> The default audio codec is currently pcmu. Opus is not currently available on our mobile infrastructure. However it will become available in Q1 of 2019. At that point the default audio codec for all 3.X mobile clients will be opus. To always use pcmu as the negotiated audio codec instead you can add it as the first codec in the `preferAudioCodecs` list.